tucson-teststand/support/docker-compose.yaml

```diff
@@ -0,0 +1,96 @@
+version: "3.6"
+
+# The different daemons are meant for different Tucson machines:
+# ospl-daemon1 - ? (master)
+# ospl-daemon2 - ?
+# ospl-daemon3 - ?
+# ospl-daemon4 - ?
```

Comment:
Consider giving the daemons more descriptive names. I think `ospl-daemon1` could be `ospl-mrster` or `ospl-main` and the others maybe have names that say something about their intention, e.g., `ospl-auxtel`, `ospl-m1m3` and so on. But I noticed you are launching a daemon on m1m3 startup script so, make sure you clear that up.

Reply:
I'll make them have the machine names like we do for the NTS and summit when I know what the machine names are. :-)